lib/wsdl.js

```diff
@@ -1112,6 +1112,12 @@ WSDL.prototype._initializeOptions = function (options) {
   } else {
     this.options.returnFault = false;
   }
+  if (options.returnNilAsNull !== undefined) {
```

Comment:
@herom please advise how you'd like me to proceed. I will update PR as you see fit.